app/models/location.rb

```diff
@@ -105,9 +102,9 @@ def as_json
 
   private
 
-  def organisation_ods_code
-    [team&.organisation&.ods_code]
-  end
+  def organisation_ods_code = [team&.organisation&.ods_code].compact
 
-  def fhir_mapper = @fhir_mapper ||= FHIRMapper::Location.new(self)
+  def fhir_mapper
+    @fhir_mapper ||= FHIRMapper::Location.new(self)
+  end
```

Comment:
Out of curiousity, why this change? Is it because the method body has `=` in it?